docs/src/spaces.md

```diff
@@ -0,0 +1,155 @@
+# [**9.** Scratchspaces](@id Scratchspaces)
+
+!!! compat "Julia 1.6"
+    Pkg's Scratchspaces functionality requires at least Julia 1.6.
+
+`Pkg` can manage and lifecycle scratchspaces of temporary or readily-recreatable data.
```

Comment:
It's not obvious to me what "lifecycle" is. Is that like garbage collect?

Reply:
Yes; lifecycle means how things get created and deleted.  Does this read better:

```
`Pkg` can manage and automatically garbage collect scratchspaces of temporary or readily-recreatable data.
```